coreutils/Makefile

```diff
@@ -410,3 +410,48 @@ exp-201609:
  		make $$program.tracerx3; \
 		make $$program.tracerx4; \
 	done
+
+# This target is used for collecting data from experiment exp-201609 result into log-exp-201609.csv file.
+csv-exp-201609:
+	rm -f log-exp-201609.csv
+	echo "Directory,Time, #instructions,#completed paths,#error paths,#subsumed paths,Average branching depth of completed paths,Average branching depth of subsumption paths,Average instructions of completed paths, Average instructions of subsumed paths,Icov,Bcov,Gcov,SLOC" >> log-exp-201609.csv; \
```

Comment:
I think we also need:
1. `#program exit paths`. I know that this number should equal `#completed paths` - `#error paths` - `#subsumed paths`, but if there were anomalies, we would like to detect it.
2. Time for actual solver calls in subsumption check (ms).
3. Number of solver calls for subsumption check.
4. Number of solver calls for subsumption check that resulted in subsumption failure.
5. Average table entries per subsumption checkpoint.
6. Average solver calls per subsumption check.